test/utils/focusVisible.ts

```diff
@@ -0,0 +1,15 @@
+import { act, fireEvent } from './createClientRender';
+
+export function dispatchFocusVisible(element: HTMLElement) {
```

Comment:
There's no focus-visible event so this shouldn't be connected to dispatch. Keeping `focusVisible` seems fine to me. 

Also: To stay in line with the rest of the codebase
```suggestion
export default function focusVisible(element: HTMLElement) {
```

Reply:
I'll fix the other references to the this function tomorrow 👌